boards/PDM/Inc/Io/Io_Aux1Aux2Efuse.h

```diff
@@ -0,0 +1,103 @@
+#pragma once
+
+#include <assert.h>
+#include <math.h>
+#include "App_Efuse.h"
+#include "Io_Efuse.h"
+#include "Io_CurrentSense.h"
+
+/**
+ * Initialize the Aux1/Aux2 Efuse
+ * @param hspi Handle to the SPI peripheral used for the Efuse
+ */
+void Io_Aux1Aux2Efuse_Init(SPI_HandleTypeDef *hspi);
+
+/**
+ * Enable the Aux 1 channel output
+ */
+void Io_Aux1Aux2Efuse_Aux1Enable(void);
```

Comment:
nit but I think we should stick to a convention here. If this function is called `Aux1Enable` then functions like `GetAux1Faults` should be renamed to `GetFaultsAux1`.

Reply:
Like Gareth said its personal preference, so up to you. Lets make it consistent though

Reply:
Ooops should be called _EnableAux1

Reply:
Okie

Reply:
I changed this